aodntools/timeseries_products/aggregated_timeseries.py

```diff
@@ -199,6 +200,7 @@ def main_aggregator(files_to_agg, var_to_agg, site_code, input_dir='', output_di
     :param opendap_url_prefix: URL prefix for OPENAP access (to be prepended to paths in files_to_agg)
     :return: name of the resulting file, list of rejected files
     """
+    files_to_agg = deepcopy(files_to_agg)
```

Comment:
:+1: Ok, this look like the right thing to do, since we're then using `files_to_agg.remove()` below to cull the list. However, 
* this doesn't seem to cause any test to fail for me (in python 3.8.13); and
* the same thing is done in [velocity_aggregated_timeseries.py](https://github.com/aodn/python-aodntools/blob/9098f6a0092289c56da2c4eba200af1e55e048ff/aodntools/timeseries_products/velocity_aggregated_timeseries.py#L87) and that doesn't fail for you?

Reply:
I haven't looked at it carefully, but I think that if you run on the list once, it'll be fine. Running it on it a second time, it might behave weird. This happens in the tests, but probably not in real life...